webpack.config.js

```diff
@@ -28,6 +28,11 @@ const commonEntryFiles = {
     background: [path.resolve(__dirname, 'src/background/background-init.ts')],
 };
 
+const electronEntryFiles = {
+    main: [path.resolve(__dirname, 'src/electron/main/main.ts')],
+    injected: [path.resolve(__dirname, 'src/injected/stylesheet-init.ts'), path.resolve(__dirname, 'src/injected/client-init.ts')],
+};
+
 const commonConfig = {
     entry: commonEntryFiles,
```

Comment:
Optional suggestion: move `entry: commonEntryFiles` into `devConfig` and `prodConfig` because it's no longer totally common (overridden in `electronConfig`).